tests/test_loaders_dumpers/input/kitchen_sink_normalized_inst_01_json.yaml

```diff
@@ -0,0 +1,41 @@
+persons:
```

Comment:
it looks like this is also identical to the yaml version -- just add one, no need for duplicates.